pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -371,9 +390,16 @@ subjectCheck:
 	}
 
 	if isUpdate {
+		if len(o.RoleBindingName) > 0 {
+			roleBinding.Name = o.RoleBindingName
```

Comment:
setting the name here is wrong... it's either a no-op or it will fail trying to mutate a name